README.md

```diff
@@ -9,83 +9,124 @@
 [![Release](https://img.shields.io/github/v/release/GIScience/openrouteservice)](https://github.com/GIScience/openrouteservice/releases/latest)
 [![LICENSE](https://img.shields.io/github/license/GIScience/openrouteservice)](LICENSE)
 
-The **openrouteservice API** provides global spatial services by consuming user-generated and collaboratively collected free geographic data directly from [OpenStreetMap](http://www.openstreetmap.org). It is highly customizable, performant and written in Java.
+Openrouteservice is a highly customizable, performant routing service written in Java. 
+It provides global spatial services by consuming user-generated and collaboratively collected free geographic data directly from [OpenStreetMap](http://www.openstreetmap.org): 
 
-The following services are available via an HTTP interface served by Tomcat.
-- **Directions** - Returns a route between two or more locations for a selected profile with customizable additional settings and instructions.
-- **Isochrones** - Obtains areas of reachability from given locations.
-- **Matrix** - Computes one-to-many, many-to-one or many-to-many routes for any mode of transport provided by openrouteservice.
+* [Directions Service](https://giscience.github.io/openrouteservice/api-reference/endpoints/directions/): Get directions for different modes of transport
+* [Isochrones Service](https://giscience.github.io/openrouteservice/api-reference/endpoints/isochrones/): Obtain areas of reachability from given locations
+* [Matrix Service](https://giscience.github.io/openrouteservice/api-reference/endpoints/matrix/): Obtain one-to-many, many-to-one and many-to-many matrices for time and distance
+* [Snapping Service](https://giscience.github.io/openrouteservice/api-reference/endpoints/snapping/): Snap coordinates to the graph edges _(/!\ not available in our live API)_
 
-To play around with openrouteservice you may use our [demonstration server](https://maps.openrouteservice.org) which comes with both the backend and a [frontend](https://github.com/GIScience/ors-map-client). Or simply [sign up](https://openrouteservice.org/dev/#/signup) for an API key and fire your requests against the API directly.
-
-Please note that openrouteservice uses a forked and edited version of [graphhopper 4.0](https://github.com/GIScience/graphhopper) which can be found [here](https://github.com/GIScience/graphhopper).
+Furthermore, there are technical endpoints that are **not available in our live API**:

```

Comment:
Make a bit clearer, that these are not part of this repo and don't come with a local installation.

Reply:
I have re-phrased it (and also re-structured it a little bit), hope it is clearer now...